discord/webhook/async_.py

```diff
@@ -163,7 +165,15 @@ async def request(
                         form_data.add_field(**p)
                     to_send = form_data
                 try:
-                    async with session.request(method, url, data=to_send, headers=headers, params=params) as response:
+                    async with session.request(
+                            method,
+                            url,
+                            data=to_send,
+                            headers=headers,
+                            params=params,
+                            proxy=proxy,
+                            proxy_auth=proxy_auth
```

Comment:
git blame tightens 🎵
```suggestion
                            proxy_auth=proxy_auth,
```